src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1395,7 +1398,7 @@ public XMPPError onSessionTerminate(JingleSession session, JingleIQ iq)
 
         synchronized (participantLock)
         {
-            terminateParticipant(participant, null, null, /* do not send session-terminate */ false);
+            terminateParticipant(participant, null, null, /* do not send session-terminate */ false, true);
```

Comment:
just for uniformity maybe add the param name in a comment before the value?

Reply:
I wish this was kotlin :)

Reply:
Soon..